### ISSUE TITLE
Browsing to a Group

### ISSUE THREAD

**Issue by thadspencer** [vincentbernat/dashkiosk#67]
RE: Usage footnote 2: _"...The appropriate token is also put in the URL in case neither cookies or local storage are available. This way, you can point the browser to the receiver part, then bookmark or turn the web page as an application."_

How do I do this, please - can I aim the browser at a URL containing a token to force the group it joins/displays?

Background: I have a chrome device in single-app-kiosk-mode, running "Chrome Sign Builder" but every time it restarts it appears in DK as a new screen.

Many Thanks

**Comment by vincentbernat** [comment]
Yes. If you cannot retrieve the final URL, open an incognito windows on your browser, go to the receiver and use the URL you get (it contains the token).

**Comment by thadspencer** [comment]
I thought that was supposed to happen but it doesn't. Even in incognito I only see "http://<fqdn>:9400/receiver" in the address bar. (i tried Chrome and Edge). Do i need to add an option to the command line to stop the full url being masked?

**Comment by vincentbernat** [comment]
Oh, you are right. When local storage is available, the token is not put in the URL. In `app/scripts/receiver/localstorage.js`, at the bottom, remove `local.setItem()`:

```js
  return {
    getItem: function(key) {
      return hash.getItem(key) || local.getItem(key) || cookies.getItem(key);
    },
    setItem: function(key, value) {
      return cookies.setItem(key, value) || hash.setItem(key, value);
    }
  };
```

This is a temporary workaround until I figure out a better method.

**Comment by thadspencer** [comment]
Thanks vb, that worked nicely :)
